cmd/chaos-builder/init.go

```diff
@@ -15,7 +15,7 @@ package main
 
 import (
 	"bytes"
-	"text/template"
+	"html/template"
```

Comment:
`chaos-builder` is a tool for code generation, we do not render HTML content.

Reply:
@STRRL so i need to revert the import in init.go and keep the file as it is or do you want me to make any more changes ?

Reply:
> so i need to revert the import in init.go and keep the file as it is

Well that file should be updated by code generation tool: protoc, you should not edit it manually.

> do you want me to make any more changes?

If you're interested in protobuf and protoc, you could take a look on `Makefile:375`, add bpfki.pb.go in that recipe.